website/integrations/services/bookstack/index.md

```diff
@@ -39,7 +35,7 @@ In authentik, under _Providers_, create a _SAML Provider_ with these settings:
 -   Signing Certificate: Choose your certificate or the default authentik Self-signed Certificate
     All other options as default.
 
-![](./authentik_saml_bookstack.png)
+![](./img/bookstack-01.png)
```

Comment:
I looked at the old and new images, and... didn;t see the difference? What changed in the actual image (not the file name)? If nothing changed, let's not change file names.